src/go/cmd/http-relay-client/client/client.go

```diff
@@ -643,8 +656,11 @@ func (c *Client) handleRequest(remote *http.Client, local *http.Client, pbreq *p
 				log.Printf("[%s] Failed to post response to relay: %v", *resp.Id, err)
 			},
 		)
-		if _, ok := err.(*backoff.PermanentError); ok {
-			// A permanent error suggests the request should be aborted.
+		// Any error suggests the request should be aborted.
+		// A missing chunk will cause clients to receive corrupted data, in most cases it is better
+		// to close the connection to avoid that.
+		if err != nil {
+			log.Printf("[%s] Closing backend connection (%s)", *resp.Id, err)
```

Comment:
You would use "%v" to log an error object.

Reply:
%w only works in fmt.Errorf, not in log.Printf. %v would be more consistent, I will fix it in a followup since I'd prefer not to block this PR any longer.